### ISSUE TITLE
FEATURE REQUEST: More than one period 

### ISSUE THREAD

**Issue by boelle** [kotope/ha_nordpool_cheapest_hours#9]
I have a small slim dishwasher than i need to run 2 times each 24 hours

would it be possible to add 2 seperate periods for the same machine?

otherwise great new version

**Comment by boelle** [comment]
@kotope will the advanced_cheapest_hours_non_sequential.yaml do this?

**Comment by kotope** [comment]
@boelle non-sequential is not suitable for this. It finds n-amount of hours, but those can be non-sequential. 
E.g. you can't get guaranteed multiple two hour slots with it.

Would need some more coding to make it happen.

**Comment by boelle** [comment]
would be nice to have that option, for now i manual edit the event created when the first one is done
